docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -120,27 +120,28 @@ Constructors
 
    Default Constructor uninitialized policy.
 
-.. cppkokkos:function:: RangePolicy(int64_t begin, int64_t end)
+.. cppkokkos:function:: RangePolicy(index_type begin, index_type end)
```

Comment:
We need to add a `versionchanged` directive: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged

Reply:
Thanks @nmm0 ! I was looking for a directive for noting a version just like that one.
But as for this line, it's a bit misleading since the argument types for this constructor were never really `int64_t`. They were templated `member_type` before which were defaulted to `int64_t`, if the policy wasn't constructed with any specified index type. So, the arg type hasn't changed, and this might be more of a correction in the docs.

Reply:
Ok makes sense, I think it's not needed then